$MirOS: ports/www/firesomething/patches/patch-security_coreconf_config_mk,v 1.2 2006/05/11 20:55:45 bsiegert Exp $
--- security/coreconf/config.mk.orig	Fri Apr 28 03:36:22 2006
+++ security/coreconf/config.mk	Tue May 20 00:24:53 2008
@@ -63,7 +63,7 @@ endif
 #######################################################################
 
 TARGET_OSES = FreeBSD BSD_OS NetBSD OpenUNIX OS2 QNX Darwin BeOS OpenBSD \
-              OpenVMS AIX
+              MidnightBSD MirBSD OpenVMS AIX
 
 ifeq (,$(filter-out $(TARGET_OSES),$(OS_TARGET)))
 include $(CORE_DEPTH)/coreconf/$(OS_TARGET).mk
